app/views/dashboard/_reports.haml

```diff
@@ -1,14 +1,21 @@
-.date_range
-  = form_tag csv_export_application_submissions_path, method: :get do
-    Export records for
-    .note_item
+.container
```

Comment:
`.container` at _this_ level makes the _content_ of the box have more margin than the others in the dashboard.